qa/L0_perf_nomodel/run_test.sh

```diff
@@ -90,14 +90,24 @@ for BACKEND in $BACKENDS; do
     MAX_LATENCY=300
     MAX_BATCH=${STATIC_BATCH} && [ $DYNAMIC_BATCH > $STATIC_BATCH ] && MAX_BATCH=${DYNAMIC_BATCH}
 
+    # openvino model does also not support batching/dynamic shapes
```

Comment:
Not all the layers support dynamic batch sizes. But I think we can still support nomodel.
See this for supported layers: https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_DynamicBatching.html#limitations

Reply:
Just fix the comment for factual accuracy. We can address the case in later PRs.

Reply:
Fixed